src/datepicker/datepicker.js

```diff
@@ -616,6 +615,11 @@ function ($compile, $parse, $document, $position, dateFilter, dateParser, datepi
 
       function validator(modelValue, viewValue) {
         var value = modelValue || viewValue;
+
+        if (!attrs.ngRequired && !value) {
```

Comment:
Yes, it works even if ngRequired is falsy. If ngRequired is falsy (i.e. anything other than ng-required=true), then the datepicker is not required. So, it considers an empty value valid in that case.